src/org/traccar/protocol/TeltonikaProtocolDecoder.java

```diff
@@ -211,11 +212,14 @@ private void decodeLocation(Position position, ChannelBuffer buf, int codec) {
 
     }
 
-    private List<Position> parseData(Channel channel, SocketAddress remoteAddress, ChannelBuffer buf) {
+    private List<Position> parseData(Channel channel, SocketAddress remoteAddress, ChannelBuffer buf,
+            boolean connectionless, int avlOffset, short packetId) {
```

Comment:
I think `avlOffset` can be renamed to just `offset`. Packet id can be `int` to avoid any explicit conversions below in the code.